1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 30 Line 18_
```markdown
$cd C://Users/aakash/Desktop
$mkdir py4wrds
$cd py4rds 
```

Create a python script called `hello_world.py` by typing into the terminal: 

`$nano hello_world.py`
```
Anaconda prompt does have nano! But we could go straight into VS Code here?